ADAPT/ISOv4Plugin
# Add local package build script

**IMPORTANT** - Please squash the commits on merge as due to the script needing checked in code I had to iterate.

I've added a script to build the solution and create the NuGet package on a local machine that is similar in function to the way the new Travis CI builds work.

### Prerequisits
- Visual Studio 2017 installed
- MS Build on the path
- NuGet on the path
- Git on the path

### Testing Steps
- Navigate PowerShell to the root of the repo on your local machine
- Check out the branch you want to test on
- Run `./package -Tag <TAG> -UseBranch $true`

### Release Steps
- Create a release on GitHub using the same pattern as Travis CI requires
- Navigate PowerShell to the root of the repo on your local machine
- Run `./package -Tag <TAG> -ApiKey <API_KEY>`
Note - If you don't provide the API Key the script will just create a package locally. You can then re-run the script to recreate and upload the package.
